packages/react-core/src/demos/DataListDemo.md

```diff
@@ -418,3 +426,353 @@ class ExpandableDataList extends React.Component {
   }
 }
 ```
+
+### Data List view
```

Comment:
I believe the corresponding demo in the HTML is simply titled `Basic`, not `Data List view`